samtools.1

```diff
@@ -1517,14 +1517,40 @@ samtools fasta
 
 Converts a BAM or CRAM into either FASTQ or FASTA format depending on the
 command invoked. The FASTQ files will be automatically compressed if the 
-filenames have a .gz or .bgzf extention.
+file names have a .gz or .bgzf extension.
+
+The input to this program must be collated by name.
+Use
+.B samtools collate
+or
+.B samtools sort -n
+to ensure this.
+
+For each different QNAME, the input records are categorised according to
+the state of the READ1 and READ2 flag bits.
+The three categories used are:
+
+1 : Only READ1 is set.
+
+2 : Only READ2 is set.
+
+0 : Either both READ1 and READ2 are set; or neither is set.
+
+For each category only one sequence will be written for a given QNAME.
+Records which include quality values will be preferred over ones that do not,
```

Comment:
This is a bit unclear. What happens if both have quality values? How exactly are the quality values compared?